clientfactory/encryptionclient.go

```diff
@@ -0,0 +1,13 @@
+package clientfactory
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3crypto"
+)
+
+//NewEncryptyionClient returns an s3crypto encryption client
+func NewEncryptyionClient(sess *session.Session, cipher s3crypto.ContentCipherBuilder) *s3crypto.EncryptionClient {
```

Comment:
Typo - s/Encryptyion/Encryption/g

Reply:
Also, does this need to be outside for mocking purposes? I only see this used in one other place, and all it does is take the parameters given and sent them through to s3crypto.NewEncryptionClient(). I don't see the benefit unless it has something to do with testing